Parsedown.php

```diff
@@ -1003,6 +1003,13 @@ public function line($text)
 
             foreach ($this->InlineTypes[$marker] as $inlineType)
             {
+                # check to see if the current inline type is nestable in the current context
+
+                if (in_array($inlineType, $non_nestables))
```

Comment:
How about `if (!empty($non_nestables) && in_array($inlineType, $non_nestables))`? `in_array()` really is *very* expensive, even with empty arrays.